lib/dropkick.js

```diff
@@ -147,7 +144,7 @@ var
     // Shallow object extend
     extend: function( obj ) {
       Array.prototype.slice.call( arguments, 1 ).forEach( function( source ) {
-        if ( source ) for ( var prop in source ) obj[ prop ] = source[ prop ];
+        if ( source ) { for ( var prop in source ) obj[ prop ] = source[ prop ]; }
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'obj'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.